### PR TITLE
37 tutorial

### PR DESCRIPTION
**Pull request by CodyGarciaa**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- tutorial walk through screen

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- modified Home/Tutorial

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- images are added as pngs, not in supabase yet -- will do that next?

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @insert pl github username here

### CODE REVIEW

File: src/navigation/BottomTabNavigator.tsx [hunk 1]
```diff
@@ -31,8 +32,8 @@ export default function NavigationBar() {
       }}
     >
       <Tab.Screen
-        name="Home"
-        component={HomeScreen}
+        name="Tutorial"
+        component={HomeNavigator}
```
Comment (angelinetu) [inline src/navigation/BottomTabNavigator.tsx:36]: Because we reworded this section to "Tutorial" from "Home", also change the naming convention for the component from "HomeNavigator" to "TutorialNavigator" and everywhere else it was previously Home.

Comment (angelinetu) [reply src/navigation/BottomTabNavigator.tsx:36]: also change *house* icon to new *book* icon in the figma

File: src/navigation/stacks/HealingResourcesNavigator.tsx [hunk 2]
```diff
@@ -73,4 +75,17 @@ export default function HealingResourcesNavigator() {
...
+    return (
+      <TouchableOpacity
+        onPress={() => navigation.goBack()}
+        style={styles.backContainer}
+      >
+        <BackArrow />
+        <Text style={styles.backText}>Healing Resources</Text>
+      </TouchableOpacity>
+    );
+  }
```
Comment (angelinetu) [inline src/navigation/stacks/HealingResourcesNavigator.tsx:90]: standardize function calls to be from "function test() {}" to "const test = (*parameters*) => {}"

File: src/navigation/stacks/LegalRightsNavigator.tsx [hunk 3]
```diff
@@ -56,4 +58,17 @@ export default function LegalRightsNavigator() {
       />
     </LegalStack.Navigator>
   );
+  function BackButton() {
```
Comment (angelinetu) [inline src/navigation/stacks/LegalRightsNavigator.tsx:61]: ^ same same

File: src/screens/Home/Tutorial/styles.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+    width: '65%',
+    bottom: 20,
+  },
+
+  button: {
+    backgroundColor: '#E6EAF1',
+    alignItems: 'center',
+    justifyContent: 'center',
```
Comment (angelinetu) [inline src/screens/Home/Tutorial/styles.ts:51]: should not be centered persay, rather align buttons to the end of the tutorial size. might require some restructuring bc right now your buttons are in their own containers.

File: src/screens/Home/Tutorial/index.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import styles from './styles';
+
+export default function Tutorial() {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const tutorialImages = [
+    require('src/assets/images/tutorial-photos/Tutorial0.png'),
+    require('src/assets/images/tutorial-photos/Tutorial2.png'),
+    require('src/assets/images/tutorial-photos/Tutorial3.png'),
```
Comment (philipye314) [inline src/screens/Home/Tutorial/index.tsx:11]: gang did ya forget how to count 🤣
also `Tutorial0.png` and `Tutorial2.png` are the same photo!

File: src/screens/Home/Tutorial/index.tsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        <TouchableOpacity
+          style={[
+            styles.button,
+            currentIndex === tutorialImages.length - 1 && styles.disabledButton,
+          ]}
+          onPress={handleNext}
+          disabled={currentIndex === tutorialImages.length - 1}
+        >
+          <Text style={styles.buttonText}>Next</Text>
+        </TouchableOpacity>
```
Comment (philipye314) [inline src/screens/Home/Tutorial/index.tsx:58]: same here

File: src/screens/Home/Tutorial/index.tsx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+      <View style={styles.tutorialBox}>
+        <Image source={tutorialImages[currentIndex]} style={styles.image} />
+      </View>
+      <View style={styles.buttonContainer}>
+        <TouchableOpacity
+          style={[styles.button, currentIndex === 0 && styles.disabledButton]}
+          onPress={handleBack}
+          disabled={currentIndex === 0}
+        >
+          <Text style={styles.buttonText}>Back</Text>
```
Comment (philipye314) [inline src/screens/Home/Tutorial/index.tsx:47]: sorry gang but kevin doesn't want a button (at all) when there's no `back` or no `next`, so instead of greying out can you take it out totally 
thanks gang

**Review by philipye314 — CHANGES_REQUESTED** [review]
mainly looks good just 2 main comments
good work angeline tu

File: src/screens/Home/Tutorial/index.tsx (hunk 5 above)
Comment (angelinetu) [reply src/screens/Home/Tutorial/index.tsx:11]: thanks king
